app/views/ReportView/components/RapidSummary/index.tsx

```diff
@@ -134,9 +132,9 @@ const RapidSummary = ({
   setIsLoading,
 }: RapidSummaryProps): JSX.Element => {
   const { report, setReport } = useContext(ReportContext);
-  const { isSigned, setIsSigned } = useContext(ConfirmContext);
+  const { setIsSigned } = useContext(ConfirmContext);
```

Comment:
this change isn't related to the filename is it?

Reply:
no this one is just part of the code cleanup so eslint will stop throwing a fit over unused vars